themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -556,10 +556,21 @@ module.controller('RealmLocalizationCtrl', function($scope, Current, $location,
 
     $scope.updateRealmSpecificLocalizationTexts = function() {
         RealmSpecificLocalizationTexts.get({id: realm.realm, locale: $scope.selectedRealmSpecificLocales }, function (updated) {
-            $scope.localizationTexts = updated;
+            $scope.localizationTexts = getSortedArrayByKeyFromObject(updated);
         })
     }
 
+    function getSortedArrayByKeyFromObject(object) {
+        const keys = Object.keys(object).sort();
+        const array = [];
+        for(let i = 0; i < keys.length; i++){
+            if(typeof object[keys[i]] === 'string') {
+                array.push([keys[i], object[keys[i]]]);
+            }
+        }
+        return array;
```

Comment:
This could possibly also be a `reduce()` operation.
```suggestion
        return keys.reduce(function (result, key) {
            return result.concat([[key, object[key]]]);
        }, []);
```

Reply:
I omitted the type check here (not sure if there are non-string values here) but if you'd want to include it you could do the following.

```ts
return keys.reduce(function (result, key) {
    const value = object[key];

    if (typeof value !== 'string') {
        return result;
    }

    return result.concat([[key, value]]);
}, []);
```